rct_image_tools/src/rct_image_tools/modified_circle_grid_finder.cpp

```diff
@@ -3,6 +3,12 @@
 #include <opencv2/calib3d.hpp>
 #include <memory>
 
+template <typename T>
+bool isEqual(const T& a, const T& b)
+{
+    return abs(a - b) < std::numeric_limits<T>::epsilon();
+}
```

Comment:
Being a little pedantic, but this function checks approximate equality. It might also be beneficial to allow a tolerance parameter in case the machine epsilon is too tight

```suggestion
template <typename T>
bool isApproxEqual(const T& a, const T& b, const T& tol = std::numeric_limits<T>::epsilon())
{
    return abs(a - b) < tol;
}
```